### PR TITLE
feat: new 'browser' option to select the rendering engine

### PR DESCRIPTION
**Pull request by sbellone**
To mitigate some flakyness that can occur with Chromium, this PR adds a new option to use a Firefox instance.
The final image size is ~1.5GB compared to ~1GB currently.

### Changes:

- Dependency changed from `playwright-chromium` to `playwright`
- New `browser` parameter in the API
- `TaskManager` now launches two browsers: `chromium` and `firefox`
  - I've removed the opening of `about://settings` which doesn't work with Firefox. Normally when `.launch()` returns, Playwright normally guarantees that the browser is up
- Updated the `/list` endpoint to return pages opened per browser

### How to test

`yarn dev`
Open http://localhost:3000 and select Firefox as a browser

### CODE REVIEW

File: src/lib/TasksManager.ts [hunk 1]
```diff
@@ -63,10 +71,13 @@ export class TasksManager {
   }
 
   async launch(): Promise<void> {
-    const browser = new Browser();
-    await browser.create();
+    const chromium = new Browser('chromium');
+    await chromium.create();
+    const firefox = new Browser('firefox');
+    await firefox.create();
```
Comment (millotp) [inline src/lib/TasksManager.ts:77]: maybe an optimization in the future would be to have 2 image of renderscript, each with it's own worker, and the crawler chooses which worker to target based on the configuration, that way we keep the image the same size.
But it's definitely a lot more work and something we can do if this ends up being an issue.

**Review by millotp — APPROVED** [review]
Huge feature ! We can try this for some time and monitor it in datadog, is renderscript hooked to some pager duty alert ?
Also could you some test that uses the `firefox` browser ? Up to you.

File: src/lib/TasksManager.ts (hunk 1 above)
Comment (sbellone) [reply src/lib/TasksManager.ts:77]: That would be perfect indeed! Let's keep it in mind.

And yes I'll add some tests with `firefox`!

**Review by millotp — APPROVED** [review]
Nice !

**Pull request merged**